data/json/monsters/civilians.json

```diff
@@ -204,7 +204,7 @@
   {
     "id": "mon_civilian_icu",
     "type": "MONSTER",
-    "name": { "str": "hospitalized human" },
+    "name": { "str": "hospitalized person" },
```

Comment:
This one should stay unchanged IMO. It's not a case of an alliterative name, but a statement. "Human" as opposed to "zombie" or some other monster.

Reply:
I changed this not only because it was an alliteration, but because it sounds rather dehumanizing. Yeah, kinda ironic calling them a human and it being dehumanizing, but I felt like it was

Reply:
IMO not dehumanizing at all and making it clear it's not a zed or some other blob entity.

Reply:
Is there anything that "hospitalized human" achieves that "hospitalized person" does not?

Reply:
Explicitly spells out it is a non-blobbed, non-zombified human

Reply:
Person does that too

Reply:
Hospitalized "human" implies there's hospitalized non-humans. This could make sense in mods that have other intelligent species, but it's redundant in vanilla. Additionally, I personally use the word "human" when referring to well, humans, _because_ it feels weird; it doesn't look as if this was the intent here.